database/pgsql/layer_test.go

```diff
@@ -18,18 +18,20 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
 	"github.com/coreos/clair/database"
 	cerrors "github.com/coreos/clair/utils/errors"
 	"github.com/coreos/clair/utils/types"
-	"github.com/stretchr/testify/assert"
 )
 
 func TestFindLayer(t *testing.T) {
-	datastore, err := OpenForTest("FindLayer", true)
+	ds, err := openDatabase(generateTestConfig("FindLayer", true))
 	if err != nil {
 		t.Error(err)
 		return
 	}
+	datastore := ds.(*pgSQL)
```

Comment:
same old story, you could maybe make this into a helper function since you use it for so many tests